editor/src/clj/editor/resource.clj

```diff
@@ -95,10 +95,36 @@
   (when-let [last-slash (string/last-index-of proj-path "/")]
     (subs proj-path 0 last-slash)))
 
+(def ^:private defignore-cache
+  ;; path->{:mtime ... :pred ...}
+  (atom {}))
+
+;; root -> pred if project path (string starting with /) is ignored
+(defn- defignore-pred [^File root]
+  (let [defignore-file (io/file root ".defignore")
+        defignore-path (.getCanonicalPath defignore-file)
+        latest-mtime (.lastModified defignore-file)
+        {:keys [mtime pred]} (get @defignore-cache defignore-path)]
+    (if (= mtime latest-mtime)
+      pred
+      (let [pred (if (.isFile defignore-file)
+                   (let [prefixes (into
+                                    #{}
+                                    (filter #(string/starts-with? % "/"))
```

Comment:
Doesn't this mean that only paths starting with `/` in the `.defignore` file will be ignored? This seems to have changed from the original behavior. I think it was using `string/contains?` previously. Was this an intended change?

Reply:
Yeah, I think it should work.